docs/ref/contrib/gis/functions.txt

```diff
@@ -237,6 +237,15 @@ The ``num_seg`` parameter is used only on PostGIS.
 Accepts a single geographic field or expression and returns the ``centroid``
 value of the geometry.
 
+``ClosestPoint``
+================
+
+.. class:: ClosestPoint(expr1, expr2, **extra)
+
+*Availability*: `PostGIS <https://postgis.net/docs/ST_ClosestPoint.html>`__, `SpatiaLite <https://www.gaia-gis.it/gaia-sins/spatialite-sql-5.0.1.html#p14b>`__
```

Comment:
Is it available in SpatiaLite 4.3.0?

Reply:
It is, I am replacing the reference URL.